test/LearningTestTool/py/kht_test.py

```diff
@@ -360,9 +360,6 @@ def evaluate_tool_on_test_dir(
             #    "Label standard out and standard error (stdout and stderr) with the rank of the process"
             if platform.system() == "Windows":
                 khiops_params.append("-l")
-            if platform.system() == "Darwin":
```

Comment:
kht_test doesn't use khiops_env, it launches the MODL binary directly. So we need this flag

Reply:
OK, I will revert this change.